Mathlib/Topology/Separation.lean

```diff
@@ -1623,16 +1623,16 @@ theorem Set.EqOn.of_subset_closure [T2Space Y] {s t : Set X} {f g : X → Y} (h
       ((hg x hx).mono_left <| nhdsWithin_mono _ hst) (h.eventuallyEq_of_mem self_mem_nhdsWithin)
 #align set.eq_on.of_subset_closure Set.EqOn.of_subset_closure
 
-theorem Function.LeftInverse.closed_range [T2Space X] {f : X → Y} {g : Y → X}
+theorem Function.LeftInverse.isClosed_range [T2Space X] {f : X → Y} {g : Y → X}
```

Comment:
can you add a deprecated alias for the old name, with a deprecation date?

Reply:
Good point, I have done with for all theorems.
I have not done this for structure fields (`OpenEmbedding.open_range` and `ClosedEmbedding.closed_range`) since I wasn't sure if that's easily possible.